docs/consume-packages/Central-Package-Management.md

```diff
@@ -0,0 +1,150 @@
+
+Central Package Management will apply to all project types – including .NET Framework – as long as compatible tooling is used.
+
+## Enabling Central Package Management
+
+To get started with central package management, you can create a `Directory.Packages.props` file at the root of your solution.
+
+Inside, you can define each of the respective package versions required of your solution using `<PackageVersion />` elements that define the package ID and version.
+
+```xml
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+  </ItemGroup>
+</Project>
+```
+
+Within a project of the solution, you can then use the respective `<PackageReference />` syntax you know and love, but without a `Version` attribute to infer the centrally managed version instead.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
+</Project>
+```
+
+Now you're using central package management and managing your versions in a central location!
+
+## Central Package Management rules
+
+The `Directory.Packages.props` file has a number of rules with regards to where it's located in a repository's directory and its context. For the sake of simplicity, only one `Directory.Packages.props` file is evaluated for a given project.
+
+What this means is that if you had multiple `Directory.Packages.props` files in your repository, the file that is closest to your project's directory will be evaluated for it. This allows you extra control at various levels of your repository.
+
+Here's an example, consider the following repository structure:
+
+```
+Repository
+ |-- Directory.Packages.props
+ |-- Solution1
+     |-- Directory.Packages.props
+     |-- Project1
+ |-- Solution2
+     |-- Project2
+```
+
+- Project1 will evaluate the `Directory.Packages.props` file in the `Repository\Solution1\` directory.
+- Project2 will evaluate the `Directory.Packages.props` file in the `Repository\` directory.
+
+## Get started
+
+To fully onboard your repository, consider taking these steps:
+
+1. Create a new file at the root of your repository named `Directory.Packages.props` that declares your centrally defined package versions in.
+2. Declare `<PackageVersion />` items in your `Directory.Packages.props`.
+3. Declare `<PackageReference />` items without `Version` attributes in your project files.
+
+<!--For an idea of how central package management may look like, refer to our [samples repo](https://github.com/NuGet/Samples/tree/main/CentralPackageManagementExample).-->
+
+## Transitive pinning
+
+You can automatically override a transitive package version even without an explicit top-level `<PackageReference />` by opting into a feature known as transitive pinning. This promotes a transitive dependency to a top-level dependency implicitly on your behalf.
```

Comment:
> This promotes a transitive dependency to a top-level dependency implicitly on your behalf

This is likely to raise a lot questions. 
Maybe clarify it further that only where the transitive dependency exists, it'll be effectively act as being promoted top level. 

I don't have a great recommendation here. 
I'll comment again if I tihnk of something.

Reply:
Yeah, I tried to simplify what this means. I think an example could help, but also maybe we just wait until there's confusion on it and fix it then.

Reply:
Does calling it a pseudo top-level dependency make it better? 
That it clearly indicates there's more to it and it's an `always like this` thing.

Reply:
Or `like` is just fine. 
The pseudo meaning in this case is the obvious one, `a pretend top-level dependency`.

Reply:
My main worry is that it's hard enough for devs to distinguish top level & transitive today. Adding the word `pseudo` or `pretend` might confuse that further?

Reply:
Fair, but misleading them could have a similar effect no? 

Maybe we change it to: 

> This promotes a transitive dependency to a top-level dependency implicitly on your behalf **when necessary**.

**Caps** is the addition.

Reply:
yeah let's do that.